src/main/java/mocacong/server/service/MemberService.java

```diff
@@ -48,4 +53,30 @@ public void delete(String email){
                 .orElseThrow(NotFoundMemberException::new);
         memberRepository.delete(findMember);
     }
+
+    public IsDuplicateEmailResponse isDuplicateEmail(String email) {
+        validateEmail(email);
+
+        Optional<Member> findMember = memberRepository.findByEmail(email);
+        return new IsDuplicateEmailResponse(findMember.isPresent());
+    }
+
+    public IsDuplicateNicknameResponse isDuplicateNickname(String nickname) {
+        validateNickname(nickname);
+
+        Optional<Member> findMember = memberRepository.findByNickname(nickname);
+        return new IsDuplicateNicknameResponse(findMember.isPresent());
+    }
+
+    public void validateEmail(String email) {
+        if (email.isBlank()) {
+            throw new InvalidEmailException();
+        }
+    }
+
+    public void validateNickname(String nickname) {
+        if (nickname.isBlank()) {
+            throw new InvalidNicknameException();
+        }
+    }
```

Comment:
접근제어자 private으로 바꿔주세요! 타 서비스나 컨트롤러에서 해당 메서드를 사용할 수 없도록
```suggestion
    private void validateEmail(String email) {
        if (email.isBlank()) {
            throw new InvalidEmailException();
        }
    }

    private void validateNickname(String nickname) {
        if (nickname.isBlank()) {
            throw new InvalidNicknameException();
        }
    }
```